htdocs/compta/paiement.php

```diff
@@ -593,7 +593,9 @@ function callForResult(imgId)
                 print '<td align="right">'.$alreadypayedlabel.'</td>';
                 print '<td align="right">'.$remaindertopay.'</td>';
                 print '<td align="right">'.$langs->trans('PaymentAmount').'</td>';
-                print '<td align="right">&nbsp;</td>';
+				$reshook=$hookmanager->executeHooks('printObjectLineTitle',$parameters,$objp,$action);
```

Comment:
$objp seems not defined.
printObjectLineTitle is a hook dedicated to show the sublines of an object (lines of an invoice, lines of an order).
May be you want to use the hood printFieldListTitle ?